packages/react/package.json

```diff
@@ -43,11 +43,18 @@
   "dependencies": {
     "@fontsource/material-icons": "5.0.1",
     "@fontsource-variable/roboto-flex": "5.0.1",
+    "@internationalized/date": "3.2.0",
```

Comment:
Can we use Intl standard api to format our dates or do we need so too early features ?

Reply:
I need date types, such as `CalendarDate` and `DateValue`, defined within the `@internationalized/date package`. These types are required in order to utilize headless implementation of the date picker's elements.